src/nbd.c

```diff
@@ -619,6 +619,8 @@ static void start_configure(struct RaucNBDContext *ctx, struct RaucNBDTransfer *
 	xfer->buffer_size = 4;
 	xfer->buffer_pos = 0;
 
+	g_debug("nbd server sending test range request to HTTP server");
```

Comment:
This is not only a test, but also used to follow redirects.

```suggestion
	g_debug("nbd server sending initial range request to HTTP server");
```